src/cloudkit.cs

```diff
@@ -2104,4 +2212,483 @@ interface CKSystemSharingUIObserver {
 		[NullAllowed, Export ("systemSharingUIDidStopSharingBlock", ArgumentSemantic.Copy)]
 		Action<CKRecordID, NSError> SystemSharingUIDidStopSharingHandler { get; set; }
 	}
+
+	[Watch (10, 0), TV (17, 0), Mac (14, 0), iOS (17, 0), MacCatalyst (17, 0)]
+	[BaseType (typeof (NSObject))]
+	interface CKSyncEngineSendChangesScope : NSCopying {
+		[NullAllowed, Export ("zoneIDs", ArgumentSemantic.Copy)]
+		NSSet<CKRecordZoneID> ZoneIds { get; }
+
+		[Export ("excludedZoneIDs", ArgumentSemantic.Copy)]
+		NSSet<CKRecordZoneID> ExcludedZoneIds { get; }
+
+		[NullAllowed, Export ("recordIDs", ArgumentSemantic.Copy)]
+		NSSet<CKRecordID> RecordIds { get; }
+
+		[Internal]
+		[Export ("initWithZoneIDs:")]
+		NativeHandle _InitWithZoneIds ([NullAllowed] NSSet<CKRecordZoneID> zoneIDs);
```

Comment:
```suggestion
		NativeHandle _InitWithZoneIds ([NullAllowed] NSSet<CKRecordZoneID> zoneIds);
```
Or is this okay with an 's' after it?